corehq/apps/api/tests.py

```diff
@@ -1233,3 +1234,31 @@ def test_paginate(self):
     def test_basic(self):
         response = self.query()
         self.assertEqual(response.status_code, 200)
+
+
+class TestApiKey(APIResourceTest):
+    """
+    Only tests access (200 vs 401). Correctness should be tested elsewhere
+    """
+    resource = v0_5.WebUserResource
+    api_name = 'v0.5'
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestApiKey, cls).setUpClass()
+        django_user = WebUser.get_django_user(cls.user)
+        cls.api_key, _ = ApiKey.objects.get_or_create(user=django_user)
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.api_key.delete()
+        super(TestApiKey, cls).tearDownClass()
+
+    def test_get_user(self):
```

Comment:
Can we also add some tests to make sure it restricts access when the user's api key is wrong (and when a user tries to use another user's api key)?